Backend/src/main/java/ktsnvt/tim1/DTOs/EventDTO.java

```diff
@@ -0,0 +1,154 @@
+package ktsnvt.tim1.DTOs;
+
+import ktsnvt.tim1.model.Event;
+import ktsnvt.tim1.model.EventCategory;
+import ktsnvt.tim1.model.EventDay;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class EventDTO {
+
+    private Long id;
+
+    @NotBlank(message = "Name cannot be empty")
+    private String name;
+
+    @NotBlank(message = "Description cannot be empty")
+    private String description;
+
+    @NotBlank(message = "Category must be specified")
+    private String category;
+
+    @NotNull(message = "Must be specified if event is active for reservations")
+    private Boolean activeForReservations;
+
+    @NotNull
+    private Boolean cancelled;
+
+    @Positive
+    private Integer reservationDeadlineDays;
+
+    @Positive
+    private Integer maxReservationsPerUser;
+
+    @Valid
+    @NotNull(message = "Event days must be specified")
+    @NotEmpty(message = "Event days must be specified")
+    private ArrayList<EventDayDTO> eventDays;
```

Comment:
@NotEmpty checks if reference is null and also if collection is empty, remove @NotNull as it is not necessary